src/core/lib/surface/init.c

```diff
@@ -180,8 +179,6 @@ void grpc_shutdown(void) {
       GRPC_EXEC_CTX_INITIALIZER(0, grpc_never_ready_to_finish, NULL);
   gpr_mu_lock(&g_init_mu);
   if (--g_initializations == 0) {
-    grpc_executor_shutdown(&exec_ctx);
-    grpc_timer_manager_set_threading(false);  // shutdown timer_manager thread
```

Comment:
This should stay. I do not see `grpc_timer_manager_set_threading(false)` being called from anyplace else.

Reply:
I think the effect of this change is to shutdown the executor and the timer_manager threads within grpc_iomgr_shutdown, which is after the plugin shutdowns.

Reply:
Yeah, but `grpc_timer_manager_set_threading` is not being called in `grpc_iomgr_shutdown`.  

So, I think we should add `grpc_timer_manager_set_threading` back in `grpc_iomgr_shutdown`

Reply:
You are right. Maybe moving the call to around `grpc_iomgr_shutdown` in here?